diff -u winsup/cygwin/include/cygwin/version.h winsup/cygwin/include/cygwin/version.h.new
--- winsup/cygwin/include/cygwin/version.h	2009-12-07 11:38:52.000000000 +0100
+++ winsup/cygwin/include/cygwin/version.h.new	2010-03-16 14:20:26.268367084 +0100
@@ -410,7 +410,8 @@
 
      /* Identifiers used in the Win32 registry. */
 
-#define CYGWIN_INFO_CYGWIN_REGISTRY_NAME "Cygwin"
+#define CYGWIN_INFO_CYGNUS_REGISTRY_NAME "Mandriva"
+#define CYGWIN_INFO_CYGWIN_REGISTRY_NAME "OpenSSH"
 #define CYGWIN_INFO_PROGRAM_OPTIONS_NAME "Program Options"
 #define CYGWIN_INFO_INSTALLATIONS_NAME   "Installations"
 
diff -u winsup/cygwin/registry.cc winsup/cygwin/registry.cc.new
--- winsup/cygwin/registry.cc	2009-10-27 12:44:20.000000000 +0100
+++ winsup/cygwin/registry.cc.new	2010-03-16 14:23:04.552180941 +0100
@@ -53,6 +53,7 @@
     }
 OK:
   new (this) reg_key (top, access, "SOFTWARE",
+		      CYGWIN_INFO_CYGNUS_REGISTRY_NAME,
 		      CYGWIN_INFO_CYGWIN_REGISTRY_NAME, NULL);
   if (top != HKEY_LOCAL_MACHINE)
     RegCloseKey (top);
diff -u winsup/cygwin/winver.rc winsup/cygwin/winver.rc.new
--- winsup/cygwin/winver.rc	2009-02-20 17:11:16.000000000 +0100
+++ winsup/cygwin/winver.rc.new	2010-03-16 14:22:03.767772628 +0100
@@ -6,6 +6,9 @@
 
 #define CYGWIN_DLL_NAME CYGWIN_VERSION_DLL_IDENTIFIER STRINGIFY(.dll)
 
+#define CYGWIN_REGISTRY_KEY CYGWIN_INFO_CYGNUS_REGISTRY_NAME "\\" \
+			    CYGWIN_INFO_CYGWIN_REGISTRY_NAME
+
 #define CYGWIN_API_VERSION STRINGIFY(CYGWIN_VERSION_API_MAJOR) "." \
 			  STRINGIFY(CYGWIN_VERSION_API_MINOR)
 
@@ -39,7 +42,7 @@
       VALUE "ProductVersion", STRINGIFY(CYGWIN_VERSION)
       VALUE "APIVersion", CYGWIN_API_VERSION
       VALUE "SharedMemoryVersion", STRINGIFY(CYGWIN_VERSION_SHARED_DATA)
-      VALUE "RegistryKey", CYGWIN_INFO_CYGWIN_REGISTRY_NAME
+      VALUE "RegistryKey", CYGWIN_REGISTRY_KEY
       VALUE "BuildDate", CYGWIN_BUILD_DATE_TIME
     END
   END
