diff -Nur ../winsup/cygwin/include/cygwin/version.h ../winsup.new/cygwin/include/cygwin/version.h
--- ../winsup/cygwin/include/cygwin/version.h	2013-08-31 20:05:31.000000000 +0200
+++ ../winsup.new/cygwin/include/cygwin/version.h	2013-10-24 10:38:46.390201600 +0200
@@ -479,7 +479,8 @@
 
      /* Identifiers used in the Win32 registry. */
 
-#define CYGWIN_INFO_CYGWIN_REGISTRY_NAME "Cygwin"
+#define CYGWIN_INFO_CYGNUS_REGISTRY_NAME "Mandriva"
+#define CYGWIN_INFO_CYGWIN_REGISTRY_NAME "OpenSSH"
 #define CYGWIN_INFO_PROGRAM_OPTIONS_NAME "Program Options"
 #define CYGWIN_INFO_INSTALLATIONS_NAME   "Installations"
 
diff -Nur ../winsup/cygwin/registry.cc ../winsup.new/cygwin/registry.cc
--- ../winsup/cygwin/registry.cc	2013-04-23 11:44:33.000000000 +0200
+++ ../winsup.new/cygwin/registry.cc	2013-10-24 10:42:34.938838400 +0200
@@ -74,6 +74,7 @@
   if (NT_SUCCESS (key_is_invalid))
     {
       new (this) reg_key ((HKEY) top, access, L"SOFTWARE",
+			  _WIDE (CYGWIN_INFO_CYGNUS_REGISTRY_NAME),
 			  _WIDE (CYGWIN_INFO_CYGWIN_REGISTRY_NAME), NULL);
       NtClose (top);
       if (key_is_invalid)
diff -Nur ../winsup/cygwin/winver.rc ../winsup.new/cygwin/winver.rc
--- ../winsup/cygwin/winver.rc	2013-06-11 09:46:40.000000000 +0200
+++ ../winsup.new/cygwin/winver.rc	2013-10-24 10:45:21.728670400 +0200
@@ -6,6 +6,9 @@
 
 #define CYGWIN_DLL_NAME CYGWIN_VERSION_DLL_IDENTIFIER STRINGIFY(.dll)
 
+#define CYGWIN_REGISTRY_KEY CYGWIN_INFO_CYGNUS_REGISTRY_NAME "\\" \
+			  CYGWIN_INFO_CYGWIN_REGISTRY_NAME
+
 #define CYGWIN_API_VERSION STRINGIFY(CYGWIN_VERSION_API_MAJOR) "." \
 			  STRINGIFY(CYGWIN_VERSION_API_MINOR)
 
@@ -39,7 +42,7 @@
       VALUE "ProductVersion", STRINGIFY(CYGWIN_VERSION)
       VALUE "APIVersion", CYGWIN_API_VERSION
       VALUE "SharedMemoryVersion", STRINGIFY(CYGWIN_VERSION_SHARED_DATA)
-      VALUE "RegistryKey", CYGWIN_INFO_CYGWIN_REGISTRY_NAME
+      VALUE "RegistryKey", CYGWIN_REGISTRY_KEY
       VALUE "BuildDate", CYGWIN_BUILD_DATE_TIME
     END
   END
