diff -Nur ../winsup/cygwin/mkvers.sh ../winsup.new/cygwin/mkvers.sh
--- ../winsup/cygwin/mkvers.sh	2013-01-21 05:38:27.000000000 +0100
+++ ../winsup.new/cygwin/mkvers.sh	2013-10-24 10:47:07.530806400 +0200
@@ -133,7 +133,7 @@
 #ifdef DEBUGGING
   "%%% Cygwin shared id: " CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version "-$builddate\n"
 #else
-  "%%% Cygwin shared id: " CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version "\n"
+  "%%% Cygwin shared id: " CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version "-mdvopenssh\n"
 #endif
   "END_CYGWIN_VERSION_INFO\n\0";
 cygwin_version_info cygwin_version =
@@ -146,7 +146,7 @@
 #ifdef DEBUGGING
   CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version "-$builddate"
 #else
-  CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version
+  CYGWIN_VERSION_DLL_IDENTIFIER "S" shared_data_version "-mdvopenssh"
 #endif
 };
 EOF
