diff --git a/binaries/data/cygwin/bin/ssh-host-config b/binaries/data/cygwin/bin/ssh-host-config
index d934d09..c1cc4c8 100755
--- a/binaries/data/cygwin/bin/ssh-host-config
+++ b/binaries/data/cygwin/bin/ssh-host-config
@@ -465,7 +465,7 @@ install_service() {
       fi
       if [ -z "${password}" ]
       then
-	if /usr/bin/cygrunsrv -I ${service_name} -d "CYGWIN ${service_name}" -p /usr/sbin/sshd \
+	if /usr/bin/cygrunsrv -I ${service_name} -d "Siveo OpenSSH agent" -p /usr/sbin/sshd \
 			      -a "-D" -y tcpip "${cygwin_env[@]}"
 	then
 	  echo
@@ -475,7 +475,7 @@ install_service() {
 	  csih_inform "will start automatically after the next reboot."
 	fi
       else
-	if /usr/bin/cygrunsrv -I ${service_name} -d "CYGWIN ${service_name}" -p /usr/sbin/sshd \
+	if /usr/bin/cygrunsrv -I ${service_name} -d "Siveo OpenSSH agent" -p /usr/sbin/sshd \
 			      -a "-D" -y tcpip "${cygwin_env[@]}" \
 			      -u "${run_service_as}" -w "${password}"
 	then
