--- /tmp/ssh-host-config	2013-10-25 15:05:53.362963124 +0200
+++ data/cygwin/bin/ssh-host-config	2013-10-25 15:05:36.107587781 +0200
@@ -625,6 +625,10 @@
 # Action!
 # ======================================================================
 
+# Create /etc/passwd and /etc/group
+chmod 755 /etc/postinstall/000-cygwin-post-install.sh.done
+/etc/postinstall/000-cygwin-post-install.sh.done
+
 # Check for running ssh/sshd processes first. Refuse to do anything while
 # some ssh processes are still running
 if /usr/bin/ps -ef | /usr/bin/grep -q '/sshd\?$'
