--- ssh-host-config	2010-10-05 15:21:24.655015239 +0200
+++ ssh-host-config.new	2010-10-05 15:22:13.531221125 +0200
@@ -643,3 +643,10 @@
   csih_inform "No ssh public key (id_dsa.pub/id_rsa.pub) found in \"${MDVCURRENTDIRPOSIX}\", \"C:\", or \"${SYSTEMDRIVE}\"."
 
 fi
+
+echo
+if cygrunsrv -S sshd; then
+  csih_inform "Mandriva OpenSSH agent started."
+else
+  csih_error "Mandriva OpenSSH agent could not be started."
+fi
