--- bin/ssh-host-config	2013-10-25 12:35:32.307782815 +0200
+++ /tmp/ssh-host-config.new	2013-10-25 12:38:00.590336611 +0200
@@ -214,12 +214,11 @@
   then
     csih_inform "Updating ${SYSCONFDIR}/sshd_config file"
     sshdconfig_tmp=${SYSCONFDIR}/sshd_config.$$
-    /usr/bin/sed -e "s/^#UsePrivilegeSeparation yes/UsePrivilegeSeparation ${privsep_used}/
+    /usr/bin/sed -i -e "s/^#UsePrivilegeSeparation yes/UsePrivilegeSeparation ${privsep_used}/
   	  s/^#Port 22/Port ${port_number}/
   	  s/^#StrictModes yes/StrictModes no/" \
-	< ${SYSCONFDIR}/sshd_config \
-	> "${sshdconfig_tmp}"
-    if ! /usr/bin/mv "${sshdconfig_tmp}" ${SYSCONFDIR}/sshd_config
+	${SYSCONFDIR}/sshd_config
+    if [ ! $? -eq 0 ];
     then
 	csih_warning "Setting privilege separation to 'yes' failed!"
 	csih_warning "Check your ${SYSCONFDIR}/sshd_config file!"
