--- bin/ssh-host-config	2013-10-25 12:46:39.579289450 +0200
+++ /tmp/ssh-host-config.new	2013-10-25 12:46:15.236182275 +0200
@@ -772,9 +772,12 @@
 
 # handle sshd_config (and privsep)
 csih_install_config "${SYSCONFDIR}/sshd_config" "${SYSCONFDIR}/defaults" || let ++warning_cnt
-if ! /usr/bin/cmp "${SYSCONFDIR}/sshd_config" "${SYSCONFDIR}/defaults/${SYSCONFDIR}/sshd_config" >/dev/null 2>&1
-then
-  /usr/bin/grep -q UsePrivilegeSeparation ${SYSCONFDIR}/sshd_config && privsep_configured=yes
+
+if [ -f "${SYSCONFDIR}/sshd_config" ]; then
+  if ! cmp "${SYSCONFDIR}/sshd_config" "${SYSCONFDIR}/defaults/${SYSCONFDIR}/sshd_config" >/dev/null 2>&1
+  then
+    grep -q UsePrivilegeSeparation ${SYSCONFDIR}/sshd_config && privsep_configured=yes
+  fi
 fi
 sshd_privsep || let warning_cnt+=$?
 
