--- ./usr/share/csih/cygwin-service-installation-helper.sh	2013-10-25 11:10:56.294978209 +0200
+++ /tmp/cygwin-service-installation-helper.sh.new	2013-10-25 12:11:23.269046246 +0200
@@ -2635,14 +2635,14 @@
     echo ""
     csih_inform "This script plans to use '${username}'."
     csih_inform "'${username}' will only be used by registered services."
-    if [ $opt_force -eq 0 ]
-    then 
-      if csih_request "Do you want to use a different name?"
-      then
-        csih_get_value "Enter the new user name:"
-        username="${csih_value}"
-      fi
-    fi
+    #if [ $opt_force -eq 0 ]
+    #then 
+    #  if csih_request "Do you want to use a different name?"
+    #  then
+    #    csih_get_value "Enter the new user name:"
+    #    username="${csih_value}"
+    #  fi
+    #fi
   else
     theservice=${opt_servicename:-the service}
     csih_inform "This script will assume that ${theservice} will run"
