--- cygwin-service-installation-helper.sh.orig	2009-03-03 09:15:29.797793915 +0100
+++ cygwin-service-installation-helper.sh	2009-03-03 09:15:42.033846999 +0100
@@ -2447,6 +2447,14 @@
       # ${username} already exists. Use it, and make no changes.
       # use passed-in value as first guess
       csih_PRIVILEGED_PASSWORD="${password_value}"
+      # Update password to match current one
+      if net user "${username}" "${password_value}" >/tmp/mdv1.$$ 2>&1; then
+        csih_inform "${username}'s password has been updated to ${password_value}."
+      else
+        csih_warning "Unable to update ${username}'s password."
+	cat /tmp/mdv1.$$
+      fi
+      rm /tmp/mdv1.$$
       return 0
     fi
 
