--- data/cygwin/bin/cyglsa-config	2013-11-14 15:27:41.186217907 +0100
+++ /tmp/cyglsa-config	2013-11-14 15:24:24.013446842 +0100
@@ -69,7 +69,9 @@
 #request "Are you sure you want to continue?" || exit 0
 
 # The registry value which keeps the authentication packages.
-value='/HKEY_LOCAL_MACHINE/SYSTEM/CurrentControlSet/Control/Lsa/Authentication Packages'
+regtool_value='/HKEY_LOCAL_MACHINE/SYSTEM/CurrentControlSet/Control/Lsa/Authentication Packages'
+regkey='HKLM\SYSTEM\CurrentControlSet\Control\Lsa'
+regvalue='Authentication Packages'
 
 [ -f /bin/cyglsa -a ! -d /bin/cyglsa ] && rm -rf /bin/cyglsa
 if [ ! -d /bin/cyglsa ]
@@ -89,7 +91,7 @@
 
 # Get old content, remove every trace of "cyglsa" from it and write the
 # content back to the registry with the new, correct path to the cyglsa DLL.
-old=`regtool get "${value}"`
+old=`regtool get "${regtool_value}"`
 
 OLDIFS=${IFS}
 IFS=$'\n'
@@ -101,7 +103,10 @@
 
 IFS=${OLDIFS}
 
-if ! regtool set "${value}" ${new} "${bindir}\\cyglsa\\${dll}"
+# Cannot use regtool here. When upgrading from an old Cygwin,
+# regtool.exe will create buggy REG_MULTI_SZ registry entity !
+#if ! regtool set "${value}" ${new} "${bindir}\\cyglsa\\${dll}"
+if ! reg add "${regkey}" /v "${regvalue}" /t REG_MULTI_SZ /d "${new}\0${bindir}\\cyglsa\\${dll}" /f
 then
   echo "Setting the new registry value failed.  Exiting."
   exit 1
